.pre-commit-config.yaml

```diff
@@ -3,47 +3,58 @@ ci:
   skip: [hadolint-docker]
 
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.0.1
+  # Autoformat: Python code
+  - repo: https://github.com/ambv/black
+    rev: 21.6b0
     hooks:
-      - id: check-yaml
-        files: .*\.(yaml|yml)$
+      - id: black
+        args: [--target-version=py36]
```

Comment:
```suggestion
        args: [--target-version=py39]
```

Reply:
I've tried and conclude there is no technical difference in the formatting on all files. If we would write it like this, would a user feel confident that they could still use python 3.8 or older? Perhaps black would autoformat it in a way that makes it not compatible?

Do you still want to change this to be py39 formatted - which practically is no difference to my knowledge?

Reply:
Ah I now understand your comment differently and see that flake version is somehow coupled with python version. Okay sure!